package/kedro_viz/models/graph.py

```diff
@@ -178,10 +178,11 @@ def create_task_node(cls, node: KedroNode) -> "TaskNode":
         Returns:
             An instance of TaskNode.
         """
+        node_name = node._name or node._func_name

```

Comment:
is this `node._name` or `node.name`? Is there any test that could demonstrate the bug on the backend?

Reply:
node._name. node.name is much longer with some extra details that are not required